code/datums/elements/weather_listener.dm

```diff
@@ -37,7 +38,7 @@
 	if(!(new_loc.z in fitting_z_levels))
 		return
 	var/datum/component/our_comp = source.AddComponent(/datum/component/area_sound_manager, playlist, list(), COMSIG_MOB_LOGOUT, fitting_z_levels)
-	our_comp.RegisterSignal(SSdcs, sound_change_signals, /datum/component/area_sound_manager/proc/handle_change)
+	our_comp.RegisterSignal(SSdcs, sound_change_signals, TYPE_PROC_REF(/datum/component/area_sound_manager, handle_change))
```

Comment:
! @AnturK Is there anything else like this that you missed?

Reply:
Ugh, probably since these have standard proc definition format. In 515 these will call declared procs too so it's pain.

Reply:
Make sure to grep it too